### ISSUE TITLE
PHPCS executable not found

### ISSUE THREAD

**Issue by jmonkfish** [JetBrains/phpstorm-workshop#41]
- OSX 10.13.3
- PhpStorm 17.3.2
- workshop/03_Inspections/06_PHPMD_PHPCS.md
- Docker branch

I was following all of the tutorials in sequence from the beginning, and ran into my first issue with this part.

> 2. Click "Create" application level copy of this deployment server and then click "OK" 

Step 1 at the top of this tutorial had no issues.  Step 2 could not be completed, I could not find "Create" anywhere, and after re-trying and snooping around, I still could not figure out this step. 

### PHPCS configuration:
> 2. Click [...] and and make sure that path for PHPCS is `phpcs`. Validate installation.

When clicking "Validate" I received the error `Cannot run program "phpcs": error=2, No such file or directory`

**Comment by MaXal** [comment]
@jmonkfish I've modified the instructions since they were indeed incorrect. Could you please check whether they are clear enough and can be followed?

**Comment by jmonkfish** [comment]
I was able to get it to work, although it seemed to require me to open up the Settings pane a few times before `phpcs` path would validate.  The first few times I tried to validate it was not able to locate PHPCS executable.  Then after a few minutes it suddenly validated.

Pretty much the same thing happened when trying to configure the Editor | Inspections to set PHP Code Sniffer to PSR2.  The Coding standard dropdown was empty.  The refresh button gave an error, as well as the `...`.  Then I returned to the PHPCS path validation, confirmed that it was still validating; then the Coding standard dropdown was populated with many standards.

Other than those two issues, everything worked great, and I'm happy to see such a quick fix for this!
